.github/workflows/admin.yml

```diff
@@ -7,26 +7,27 @@ on:
       - release-*
     tags:
       - v[0-9]+.[0-9]+.[0-9]+*
+
   pull_request:
     paths-ignore:
       - "agent/**"
       - "managed/**"
       - "docs/**"
 
+  workflow_call:
+    inputs:
+      go-version:
+        required: false
+        type: string
+        default: tip
+
 jobs:
   test-admin:
     name: Test pmm-admin
     runs-on: ubuntu-22.04
-    strategy:
-      fail-fast: false
-      matrix:
-        go-version: [ 1.18.x ]
-        may-fail: [ false ]
-    #        include:
-    #          - go-version: tip
-    #            may-fail: true
```

Comment:
I think better to keep matrix. Let's try this approach: https://stackoverflow.com/questions/69781005/combine-dynamic-github-workflow-matrix-with-input-values-and-predefined-values

Reply:
Why do you think we need matrix if we don't use matrix-based workflows? Also there is no any answer at the link you provided. Is there any working example of the approch you described?

Reply:
It will be nice to have matrix because it's easy to add new values during migration period. But yes, it seems that there is no working solution for that.

Reply:
I added this ability.